config.js

```diff
@@ -84,9 +84,9 @@ config.orionDb = {
  * at core. Each <<interval>> ms, the rules are sent to core.
  */
 config.perseoCore = {
-    rulesURL: 'http://localhost:8080/perseo-core/rules',
-    noticesURL: 'http://localhost:8080/perseo-core/events',
-    interval: 60e3 * 5
+    rulesURL: 'http://localhost:8080/perseo-main/rules',
+    noticesURL: 'http://localhost:8080/perseo-main/events',
```

Comment:
I'm not sure about this change:
API still uses it: https://github.com/telefonicaid/perseo-core/blob/master/documentation/api.md
As well ass docker env var: https://github.com/telefonicaid/perseo-fe/blob/master/bin/perseo#L115